aws/cloudfront/invalidation/lambda/index.js

```diff
@@ -53,21 +54,22 @@ exports.handler = (event, context, callback) => {
     			
     			var arn = record.eventSourceARN.split(':', 6);
                 var queueUrl = 'https://sqs.'+ arn[3] +'.amazonaws.com/'+ arn[4] +'/'+ arn[5];
-                
+
+                // This Message must be in SNS => SQS Format for incoming parsing to work
     			var params = {
-    				MessageBody: JSON.stringify(message),
+    				MessageBody: JSON.stringify({Message: JSON.stringify(message)}),
     				QueueUrl: queueUrl,
-    				DelaySeconds: 320
+    				DelaySeconds: RETRY_TIMOUT
     			};
-			
+			    var msg = `[RETRY] retrying ${invalidationParams.DistributionId}:${invalidationParams.Paths}`
 	    		console.log(params)
 
     			sqs.sendMessage(params, function (err, data) {
     				if (err) {
     					console.log(err);
     					callback("failed to send message for retry");
     				} else {
```

Comment:
Don't need this else block since the else block resolves with the callback

```
if (err) {
  console.log(err)
  callback(...);
}

callback(null, `[RETRY] Scheduling a retry in ${RETRY_TIMOUT} seconds. ${retried} retries`)
```

Reply:
I was hoping someone with better js skills could clean up my mess. Probably better if I started actually learning this stuff.